pkg/config/centralconfig.go

```diff
@@ -516,6 +571,12 @@ func AddCentralConfigProperties(props properties.Properties, agentType AgentType
 
 	if agentType == TraceabilityAgent {
 		props.AddStringProperty(pathDeployment, "prod", "AMPLIFY Central")
+		props.AddStringProperty(pathLighthouseURL, "https://lighthouse-staging.admin.staging.appctest.com", "URL of the Lighthouse")
+		props.AddStringProperty(pathPlatformEnvironmentID, "", "Platform Environment ID")
+		props.AddBoolProperty(pathPublishUsage, true, "Indicates if the agent can publish usage event to AMPLIFY platform. Default to true")
+		// props.AddBoolProperty(pathPublishMetric, true, "Indicates if the agent can publish metric event to AMPLIFY platform. Default to true")
+		props.AddBoolProperty(pathPublishTraffic, true, "Indicates if the agent can publish traffic event to AMPLIFY platform. Default to true")
+		props.AddDurationProperty(pathEventAggregationInterval, 1*time.Minute, "The time interval at which usage and metric event will be generated.")
```

Comment:
May be we should increase this default time to 5 or 10 mins?

Reply:
fixed